examples/allennlp/allennlp_jsonnet.py

```diff
@@ -20,17 +20,21 @@
 
 """
 
+import json
 import os.path
 import shutil
 
 import optuna
+from optuna.integration.allennlp import dump_best_config
 from optuna.integration import AllenNLPExecutor
 
 
 # This path trick is used since this example is also
 # run from the root of this repository by CI.
 EXAMPLE_DIR = os.path.dirname(os.path.abspath(__file__))
+CONFIG_PATH = os.path.join(EXAMPLE_DIR, "classifier.jsonnet")
 MODEL_DIR = "result"
+BEST_CONFIG_PATH = os.path.join(MODEL_DIR, "best.classifier.json")
```

Comment:
This example shows the content of `best.classifier.json` in the stdout because the `best.classifier.json` will be removed with its parent directory. But, I feel it would be easy to understand the usage of `dump_best_config` if it keeps the file and shows the file path.
How about placing it in the current directory?

Reply:
I agree with you, I revised PR to create `best.classifier.json` on current directory in 2400ca3.

```
%python allennlp_json.py
....
....
[I 2020-05-14 23:57:53,253] Finished trial#2 with value: 0.56328 with parameters: {'DROPOUT': 0.38850725212419135, 'EMBEDDING_DIM': 46, 'MAX_FILTER_SIZE': 3, 'NUM_FILTERS': 32, 'HIDDEN_SIZE': 21}. Best is trial#2 with valu
e: 0.56328.
Number of finished trials:  3
Best trial:
  Value:  0.56328
  Params:
    DROPOUT: 0.38850725212419135
    EMBEDDING_DIM: 46
    MAX_FILTER_SIZE: 3
    NUM_FILTERS: 32
    HIDDEN_SIZE: 21

Created optimized AllenNLP config to `best.classifier.json`
```